aggregator/src/test/java/com/redhat/cloud/notifications/helpers/ResourceHelpers.java

```diff
@@ -33,9 +33,9 @@ public void purgeEmailAggregations() {
 
     @Transactional
     public Integer purgeOldAggregation(EmailAggregationKey key, LocalDateTime lastUsedTime) {
-        String query = "DELETE FROM EmailAggregation WHERE accountId = :accountId AND bundleName = :bundleName AND applicationName = :applicationName AND created <= :created";
+        String query = "DELETE FROM EmailAggregation WHERE orgId = :orgId AND bundleName = :bundleName AND applicationName = :applicationName AND created <= :created";
         return session.createQuery(query)
-                .setParameter("accountId", key.getAccountId())
+                .setParameter("orgId", key.getOrgId())
```

Comment:
Does all of our current aggregations have the `orgId` set? Just checking we are not ending up with data that will never go away (although we can check later if we have old data)

Reply:
Yes they do. The `org_id` field was changed to `NOT NULL` in https://github.com/RedHatInsights/notifications-backend/pull/1428.